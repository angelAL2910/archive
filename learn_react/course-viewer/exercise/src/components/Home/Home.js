import React from "react";
import { Link } from "react-router-dom";
import "../../App.css";

class Home extends React.Component {
  render() {
    return (
      <div className="element">
        <div className="jumbotron">
          <h1 className="display-4">Pluralsight Administration</h1>
          <p className="lead">
            React, Redux and React Router for ultra-responsive web apps.
          </p>
          <Link className="btn btn-primary btn-lg" to="/about" role="button">
            Learn more
          </Link>
        </div>
      </div>
    );
  }
}

export default Home;
