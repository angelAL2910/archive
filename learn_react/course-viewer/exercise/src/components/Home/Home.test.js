import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import Home from "./Home";

test("Tests for the About component", () => {
        let tree = renderer.create(
            <BrowserRouter>
                <Home />
            </BrowserRouter>
        ).toJSON()

        expect (tree).toMatchSnapshot();
})