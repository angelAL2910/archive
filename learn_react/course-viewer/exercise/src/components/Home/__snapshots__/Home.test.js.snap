// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the About component 1`] = `
<div
  className="element"
>
  <div
    className="jumbotron"
  >
    <h1
      className="display-4"
    >
      Pluralsight Administration
    </h1>
    <p
      className="lead"
    >
      React, Redux and React Router for ultra-responsive web apps.
    </p>
    <a
      className="btn btn-primary btn-lg"
      href="/about"
      onClick={[Function]}
      role="button"
    >
      Learn more
    </a>
  </div>
</div>
`;
